Plots/Animations/NCL_animate_1.py

```diff
@@ -92,8 +92,8 @@ def animate(i):
         ylabel="")
 
 
-# runs the animation initiated with the frame from init and progressed with the animate function
+# Iterates over the animation initiated with the frame int and progressed with the animate function
```

Comment:
"Iterates over" is better wording, but the phrase "from init" refers to the initialization of the plot, not an int. Its shortened as init because that is the commonly used shortening in this context from the keyword argument in the FuncAnimation call that is called init. But as this example actually got rid of the explicit init, I can see the confusion. May I suggest the following rephrase:

```suggestion
# Iterates over the initiated animation initiated when progressed with the animate function
```